app/mailers/maintenance.rb

```diff
@@ -7,9 +7,12 @@ def account_removal_warning(user)
     @pod_url = AppConfig.environment.url
     @after_days = AppConfig.settings.maintenance.remove_old_users.after_days.to_s
     @remove_after = @user.remove_after
-    mail(:to => @user.email, :subject => I18n.t('notifier.remove_old_user.subject')) do |format|
-      format.text
-      format.html
+
+    I18n.with_locale(@user.language) do
+      mail(:to => @user.email, :subject => I18n.t('notifier.remove_old_user.subject')) do |format|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.